qml/items/PlayField.qml

```diff
@@ -19,134 +19,114 @@ Item {
     property var selectedCard : null
     property var db
 
-    Cell {
```

Comment:
Factored "type" into ClassName.
Delegated setting "maxStack" to specific subclass.